discovery-provider/src/tasks/entity_manager/entities/track.py

```diff
@@ -243,7 +240,7 @@ def validate_track_tx(params: ManageEntityParameters):
                 raise IndexingValidationError(
                     f"Track {track_id} description exceeds character limit {CHARACTER_LIMIT_TRACK_DESCRIPTION}"
                 )
-    else:
+    if params.action == Action.UPDATE or params.action == Action.DELETE:
```

Comment:
these validations are only being applied for DELETE when they should work for UPDATE as well